rts/lra/pom.xml

```diff
@@ -36,6 +36,9 @@
 
         <test.logs.to.file>false</test.logs.to.file>
         <skipITs>true</skipITs> <!-- integration tests are disabled by default, use profile to run 'arq' -->
+
+        <thorntail.test.logging></thorntail.test.logging>
+        <java.util.file.logging.level></java.util.file.logging.level>
```

Comment:
please, remove this property completely. From what I know (if I'm not badly mistaken somehow) the value of `ConsoleHandler` has first no impact to be passed as `-D` command line property. Second has for sure no impact to thorntail logging setup.

Reply:
okay